Sources/networkcommunication/ALRequestHandler.m

```diff
@@ -125,7 +125,7 @@ + (void)addGlobalHeader:(NSMutableURLRequest *)request
     }
     
     if (userId) {
-        [request setValue:[userId urlEncodeUsingNSUTF8StringEncoding] forHTTPHeaderField:@"Of-User-Id"];
+        [request addValue:userId forHTTPHeaderField:@"Of-User-Id"];
```

Comment:
@amntoppo  as per this change, we will not encode the userid for any of the APIs.Are you sure it won't affect other apis? Have you confirmed with FF team?

Reply:
yes, server is not decoding this header so if the userid is an email, the server call fails.
I have confirmed with Adarsh sir and also tested in different APIs which are sending this header.